### ISSUE TITLE
Make the voting function

### ISSUE THREAD

**Issue by aaspinwall** [aaspinwall/collab#41]
Take [this video](https://www.youtube.com/watch?v=orybDrUj4vA) as a reference and code it up. It should work as a utility function for the backend.

The function should take an object with the choices and votes inside: 

```js
const choices = {
    pizza: ["Tom", "Jane", "Simon", "Tiger Boy"],
    poutine: ["Jane", "Simon", "Tiger Boy"],
    chicken: ["Jane"],
};

function(choices) {
const response =    {
        tied: //boolean: is there a tie?
        winner: //winner: if no winner, null
        ranked: //choices sorted by votes
    }
    return response
}
}
//ps this is mock code, don't use it as is
```
Each participant gets to vote as many times as they want.
The function should return an object with the results (see above)

**Comment by NATHALIEBENARROCH** [comment]
I would like to try this one

**Comment by aaspinwall** [comment]
> I would like to try this one

You go for it

**Comment by aaspinwall** [comment]
This issue is open again. Feel free to comment so we can assign it to you

**Comment by AlexandreGagnon-Lalonde** [comment]
Let me attack this one now!

**Comment by aaspinwall** [comment]
> Let me attack this one now!

You got it!

**Comment by Edu93Jer** [comment]
Hi @aaspinwall if @AlexandreGagnon-Lalonde is not working on the issue anymore, do you think I can go for it?

**Comment by AlexandreGagnon-Lalonde** [comment]
Hey @Edu93Jer I actually finished the function, but if you want to have a go and find a better solution than mine go ahead!

**Comment by AlexandreGagnon-Lalonde** [comment]
You can look at the last PR I made on this project

**Comment by Edu93Jer** [comment]
Oh, np I saw the issue open, and without a PR pointing at the issue, so that's why I asked, but if you already did this is fine.

**Comment by aaspinwall** [comment]
Sorry, guys. I forgot to close it. Yes it was closed on #110 
@Edu93Jer would you be interested in working on #73 ?

**Comment by aaspinwall** [comment]
@Edu93Jer we do need help on #73 if you're interested. Sharing the link on social media is still to be implemented.
